### ISSUE TITLE
ModuleGenerator in launch clamps not taken into account by PlannerController when OUTPUT_RESOURCE ElectricCharge not on last position (or the only one)

### ISSUE THREAD

**Issue by Gordon-Dry** [Kerbalism/Kerbalism#666]
As soon as the ModuleGenerator got more than one OUTPUT_RESOURCE - even if ElectricCharge is topmost in the MM cache - the PlannerController does not take it into account anymore when toggled OFF in VAB with "Simulate generator in planner".
It always shows incoming EC from the launch clamp that is affected.

**Comment by Gordon-Dry** [comment]
btw this seems to be the reason for this written to the log:
```[LOG 21:04:54.161] [Kerbalism] ResourceSimulator.Process_apiModule resource count before call 0
[LOG 21:04:54.161] [Kerbalism] ResourceSimulator.Process_apiModule resource count after call 1```

Always dozens of those pair of lines.

**Comment by Gordon-Dry** [comment]
Edit:
When  taken OFF because the planner always shows the EC coming from the generator in the launch clamps that are affected.

**Comment by Gordon-Dry** [comment]
When ElectricCharge is the last entry, it works.
Like in this example:
```
		MODULE
		{
			name = ModuleGenerator
			requiresAllinputs = false
			resourceThreshold = 0.001
			activateGUIName = Fuel On
			shutdownGUIName = Fuel Off
			isAlwaysActive = true
			OUTPUT_RESOURCE
			{
				name = LiquidFuel
				rate = 45
			}
			OUTPUT_RESOURCE
			{
				name = Oxidizer
				rate = 55
			}
			OUTPUT_RESOURCE
			{
				name = MonoPropellant
				rate = 25
			}
			OUTPUT_RESOURCE
			{
				name = LqdHydrogen
				rate = 825
			}
			OUTPUT_RESOURCE
			{
				name = ElectricCharge
				rate = 1
			}
		}

```

**Comment by gotmachine** [comment]
The solution is to use an actual Kerbalism process for launch clamps.
This will be implemented in 4.0
